docs/enhancements/cluster_pool_hot_spares.md

```diff
@@ -0,0 +1,162 @@
+- Manually resuming a pool CD will not cause it to be claimed earlier.
+  In fact, it is subject to being re-hibernated by the controller.
+  This is an acceptable design point: You can manage power states by configuring your ClusterPool; don't expect to be able to do so manually.
+
+### Hibernation Controller Must Account For `hibernateAfter` in Pool Clusters
+
+If the ClusterPool has `hibernateAfter` set, that clock starts on a given ClusterDeployment from the time its
+[Hibernating condition is Running](https://github.com/openshift/hive/blob/28f67bab3dac5fb9dec405e0536d64187faf964d/pkg/controller/hibernation/hibernation_controller.go#L240-L244);
+or, if the status of that condition is Unknown, from
+[when installation completed](https://github.com/openshift/hive/blob/28f67bab3dac5fb9dec405e0536d64187faf964d/pkg/controller/hibernation/hibernation_controller.go#L229-L239).
+If we don't account for this, Running unassigned CDs in the pool will hibernate, and then the [new logic](#decouple-powerstate-on-creation) will immediately wake them back up, resulting in unnecessary churn and potential claim delays.
+So we'll:
+- Add another Condition to ClusterDeployment called `Claimed`, indicating claim assignment status.
+  It is:
+  - `Unknown` for CDs not part of a pool, permanently.
+  - Initialized and maintained as `False` for unassigned pool CDs.
+  - Set to `True` when a CD is assigned to a claim. This should be terminal: once `True`, it should never change again.
+- Add logic to the hibernation controller as follows:
+  - If the CD is part of a ClusterPool
+    - If it is unclaimed, ignore `hibernateAfter`* -- never hibernate.
+    - If it is claimed, use as the base time the _later_ of
+      - the `lastTransitionTime` of the new `Claimed` Condition
+      - the existing algorithm's base time (this accounts for hibernating/resuming during the normal post-claim lifecycle of the CD).
+  - Else (not part of a ClusterPool) do what we do today.
+
+*This also means a manually-resumed unclaimed CD from a pool will never hibernate at the behest of `hibernateAfter`.
+As noted [above](#clusterclaim-assignment-by-creationtimestamp), you shouldn't be trying to manage the PowerState of pool-owned CDs manually.
+This goes double if you're using `hibernateAfter` with ClusterPools.
+
+## Alternatives
+
+The following alternatives were considered:
+
+### Don't Hibernate New ClusterDeployments
+
+Today when ClusterDeployments are created to fill out the pool, they are created with `PowerState = HibernatingClusterPowerState`.
+Add an option to ClusterPool to not do that.
+
+**Pros:**
+- Dead simple
+
+**Cons:**
+- In periods of low activity, you're burning money by having Running vs. Hibernating clusters.
+  (I suppose you could manually hibernate clusters in the pool. Not the best UX.)
+
+### `runningPercent`
+
+Use a percentage instead of an absolute number to indicate how many clusters in the pool should remain Running.
```

Comment:
the kubeapis have IntOrString type that allows users to decide whether they want to use exact numbers or percentages.
for example see https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/deployment-v1/#DeploymentSpec `strategy.rollingUpdate.maxSurge`

Reply:
That's neat. Though (Gurney and) I feel the reasons for not choosing to use a percentage stand on their own, beyond the logistics of having to deal with the data types.